App/Reducers/LoginReducer.js

```diff
@@ -9,20 +9,24 @@ export const INITIAL_STATE = Immutable({
 })
 
 // login attempts
-const attempt = (state, action) =>
-  state.merge({ attempting: true })
+const attempt = (state, action) => {
+  return { ...state, ...{ attempting: true } }
```

Comment:
My guess is that the `autoRehydrate()` mtehod returns back a state object that's different than it used to without the `merge` mixin applied.